piet-web/src/lib.rs

```diff
@@ -262,6 +262,13 @@ impl<'a> RenderContext for WebRenderContext<'a> {
         let _ = self.ctx.transform(a[0], a[1], a[2], a[3], a[4], a[5]);
     }
 
+    fn current_transform(&self) -> Affine {
+        // todo
+        // current_transform() and get_transform() currently not implemented:
+        // https://github.com/rustwasm/wasm-bindgen/blob/f8354b3a88de013845a304ea77d8b9b9286a0d7b/crates/web-sys/webidls/enabled/CanvasRenderingContext2D.webidl#L136
+        Affine::default()
```

Comment:
I did open a PR to get this added: https://github.com/rustwasm/wasm-bindgen/pull/1966
Not sure what the best way here is, wait until it's merged or return something like `Affine::default()` for now and change it later?

Reply:
yea this looks good. I would also open an issue when this is merged.

Reply:
https://github.com/rustwasm/wasm-bindgen/pull/1966#issuecomment-577242039 got merged, now we'll just have to wait until a new version gets released to use it